crates/ingress-rpc/src/service.rs

```diff
@@ -214,7 +214,12 @@ where
             .map_err(|e: String| EthApiError::InvalidParams(e).into_rpc_err())?;
 
         let bundle_hash = &parsed_bundle.bundle_hash();
-        let meter_bundle_response = self.meter_bundle(&bundle, bundle_hash).await?;
+        let meter_bundle_response = self.meter_bundle(&bundle, bundle_hash).await
+            .unwrap_or_else(|_| {
+                // TODO: in the future, we should return the error
+                warn!(message = "Bundle simulation failed, using default response", bundle_hash = %bundle_hash);
+                MeterBundleResponse::default()
```

Comment:
maybe `default_with_hash()` to use the bundle hash here? not sure if it matters

but also dont want to make this fn look like something to frequently use